Purchases/Purchasing/RCAttributionFetcher.m

```diff
@@ -27,12 +27,33 @@ + (instancetype)sharedManager;
 
 @implementation RCAttributionFetcher : NSObject
 
-- (nullable NSString *)advertisingIdentifier
+- (NSString *)rotateString:(NSString *)string
+{
+    NSMutableString *rotatedString = [NSMutableString string];
+    for (NSUInteger charIdx = 0; charIdx < string.length; charIdx++) {
+        unichar c = [string characterAtIndex:charIdx];
+        unichar i = '0';
+        if (('a' <= c && c <= 'm') || ('A' <= c && c <= 'M')) {
+            i = (unichar) (c + 13);
```

Comment:
it'd be nice to name rot13 somewhere, in case anyone wonders why this implementation. maybe a comment or a method name